cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
I'm not crazy about the naming here, if it's being used to indicate that the restore first needs to run a truncate ideally that would be the first job ran with a different type.

Something like this:
1. Run `NodeJobTask { jobType = TRUNCATE }`
2. foreach node in nodes `NodeJobTask { jobType = RESTORE, restoreFile = "/local/path/to/backup.bak" }`

Let's keep the smarts about what has to happen in the scheduler which can coordinate the tasks, rather than having tasks figure out some special things based on the global state or other stuff.